### PR TITLE
Added vecorization user documentation

### PR DESCRIPTION
**Pull request by harshagrwl**

### CODE REVIEW

**Comment by harshagrwl** [comment]
@morevnaproject Please review this

File: source/artwork_import/vectorization.rst [hunk 1]
```diff
@@ -1,3 +1,45 @@
...
+The Algorithm of Converting Vector Graphics from Bitmaps consists of the following steps:
+    * **Polygonization**: A Polygon is extracted from the image which gives a minimal representation of image contours.
+    * **Skeletonization**: The image is effectively reduced to an image of lines and the essential structure of the image is not altered.
+    * **Convert to strokes**: Conversion of the sequence into strokes. `Hermite Curves <https://www.inf.ed.ac.uk/teaching/courses/cg/d3/hermite.html>`_ are used for creation of strokes.
+
+
+
+Usage
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Upload the image and right-click on the Image Layer and select "Convert to Vector" command.4
```
Comment (morevnaproject) [inline source/artwork_import/vectorization.rst]: I think it's better to remove "4" at the end of line

Comment (morevnaproject) [inline source/artwork_import/vectorization.rst]: I think it's better to remove "4" at the end of line.

File: source/artwork_import/vectorization.rst [hunk 2]
```diff
@@ -1,3 +1,45 @@
-########################
-    Vectorization
-########################
+Vectorization of Bitmaps
+==============================================
+
+Synfig Studio comes with a feature of Vectorization of Bitmaps. In this tutorial, we will learn what is Vectorization feature and how to use it. :)
+
+What is Vectorization of Bitmaps?
```
Comment (morevnaproject) [inline source/artwork_import/vectorization.rst]: I suggest to change to "What is Vectorization means?"

File: source/artwork_import/vectorization.rst [hunk 3]
```diff
@@ -1,3 +1,45 @@
...
+There are two common ways to represent a two-dimensional image:
+
+    * Raster Graphics or Bitmap Graphics
+    * Vector Graphics
+
+In Bitmap Graphics images are stored as a two-dimensional grid of pixels. This has some problems for example if a bitmap image is scaled it becomes blurry and pixelated.
+
+We use Vector Graphics to solve this problem. For converting Bitmap image to Vector Image, Synfig uses vectorization algorithm from `OpenToonz <https://opentoonz.github.io/e/>`_
+
+Algorithm
```
Comment (morevnaproject) [inline source/artwork_import/vectorization.rst]: I suggest to remove "Algorithm" section, as this information is not intended for user.

File: source/artwork_import/vectorization.rst [hunk 4]
```diff
@@ -1,3 +1,45 @@
...
+    * **Skeletonization**: The image is effectively reduced to an image of lines and the essential structure of the image is not altered.
+    * **Convert to strokes**: Conversion of the sequence into strokes. `Hermite Curves <https://www.inf.ed.ac.uk/teaching/courses/cg/d3/hermite.html>`_ are used for creation of strokes.
+
+
+
+Usage
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Upload the image and right-click on the Image Layer and select "Convert to Vector" command.4
+
+    .. image:: ../images/convert_to_vector.png
```
Comment (morevnaproject) [inline source/artwork_import/vectorization.rst]: Image is missing.

File: source/artwork_import/vectorization.rst [hunk 5]
```diff
@@ -1,3 +1,45 @@
...
+
+Clicking on this "Convert to Vector" will open a dialog box with the following settings:
+    
+    .. image:: ../images/vector_dialog_box.png
```
Comment (morevnaproject) [inline source/artwork_import/vectorization.rst]: This image is missing too.

File: source/artwork_import/vectorization.rst (hunk 2 above)
Comment (rodolforg) [reply source/artwork_import/vectorization.rst]: *What does Vectorization mean? 
=)

**Comment by harshagrwl** [comment]
@morevnaproject I have resolved all the above-mentioned issues in the latest commit.

File: source/artwork_import/vectorization.rst (hunk 2 above)
Comment (morevnaproject) [reply source/artwork_import/vectorization.rst]: Right =D

**Comment by morevnaproject** [comment]
Looks good now. Thank you!

**Pull request merged**